purchases/src/customEntitlementComputation/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -224,16 +224,24 @@ class Purchases internal constructor(
         /**
          * Configures an instance of the Purchases SDK with a specified API key. The instance will
          * be set as a singleton. You should access the singleton instance using [Purchases.sharedInstance]
-         * @param configuration: the [PurchasesConfiguration] object you wish to use to configure [Purchases].
+         * @param context: the Application context object of your Application.
+         * @param apiKey: the API Key for your app. Obtained from the RevenueCat dashboard.
+         * @param appUserID: a unique id for identifying the user.
          * @return An instantiated `[Purchases] object that has been set as a singleton.
          */
         @JvmStatic
-        fun configure(
-            configuration: PurchasesConfiguration,
+        fun configureInCustomEntitlementsComputationMode(
```

Comment:
This means that in this mode, anything else that can be enabled in `PurchasesConfiguration` won't be available, like setting a custom executorService for requests, observer mode, entitlement verification mode.... This is ok for now, but I wanted to mention it.